R/as_polars.R

```diff
@@ -367,13 +369,15 @@ as_polars_series.POSIXlt = function(x, name = NULL, ...) {
 #' @rdname as_polars_series
 #' @export
 as_polars_series.data.frame = function(x, name = NULL, ...) {
-  pl$DataFrame(unclass(x))$to_struct(name = name)
+  as_polars_df(x)$to_struct(name = name)
```

Comment:
This change is not directly related, but is a small refactoring to make object conversions more appropriate.